### PR TITLE
Small things

### PR DESCRIPTION
**Pull request by ChunyiLyu**
# Changes

- :broom: Small code style refactors that I noticed while looking at the repo, commit messages should be self explanatory. Unit tests are all passing locally. Could not get conformance tests to run sadly :/

/kind cleanup

**Release Note**

```release-note
Small code refactors
```

**Docs**

### CODE REVIEW

**Comment by gabo1208** [comment]
What errors are you facing when trying to run conformance tests locally?

File: pkg/reconciler/testing/factory.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func MakeFactory(ctor Ctor, unstructured bool, logger *zap.SugaredLogger) Factor
 		ctx = controller.WithEventRecorder(ctx, eventRecorder)
 
 		// Configure the ConfigMap static watcher
-		configMaps := []*corev1.ConfigMap{}
+		var configMaps []*corev1.ConfigMap
```
Comment (embano1) [inline pkg/reconciler/testing/factory.go:94]: 👍

**Review by embano1 — APPROVED** [review]
LGTM

**Comment by ChunyiLyu** [comment]
@gabo1208 

Yes I tried `make test-conformance` locally this is the error I kept on getting:
```
BROKER_TEMPLATES=/Users/clyu/workspace/eventing-rabbitmq/test/conformance/testdata/with-operator \
	BROKER_CLASS=RabbitMQBroker \
	go test -v -tags=e2e \
		-count=1 -parallel=12 -timeout=20m \
		-run TestBrokerConformance /Users/clyu/workspace/eventing-rabbitmq/test/conformance/...
{"level":"info","ts":1642430963.129856,"logger":"fallback","caller":"injection/injection.go:61","msg":"Starting informers..."}
level All, state Any, feature

=== RUN   TestBrokerConformance
2022/01/17 14:49:24 Using base gcr.io/distroless/static:nonroot for knative.dev/reconciler-test/cmd/eventshub
2022/01/17 14:49:26 Building knative.dev/reconciler-test/cmd/eventshub for linux/amd64
2022/01/17 14:49:30 Loading kind.local/eventshub:7f87812e19d9044cf261237a73736c08b5d8a2b297018aa77d5250786042f734
Error: failed to publish images: error publishing ko://knative.dev/reconciler-test/cmd/eventshub: no nodes found for cluster "kind"
2022/01/17 14:49:30 error during command execution:failed to publish images: error publishing ko://knative.dev/reconciler-test/cmd/eventshub: no nodes found for cluster "kind"
error attempting to ko publish: exit status 1
--- FAIL: TestBrokerConformance (7.73s)
panic: exit status 1 [recovered]
	panic: exit status 1
```

I do have a local kind cluster created and targeted. Is there some other setup that I missed?

**Comment by gabo1208** [comment]
How is your local cluster called? If it is not called kind try settting the KIND_CLUSTER_NAME VARIABLE to the cluster's name

**Comment by ChunyiLyu** [comment]
@gabo1208 Yes my kind cluster is called `kind`. I set `KIND_CLUSTER_NAME_VARIABLE` to `kind` just in case, but it's the same failure.

**Comment by benmoss** [comment]
> @gabo1208 Yes my kind cluster is called `kind`. I set `KIND_CLUSTER_NAME_VARIABLE` to `kind` just in case, but it's the same failure.

No, he meant `KIND_CLUSTER_NAME`. An unfortunate choice in the `Makefile` is that [it sets that variable](https://github.com/ChunyiLyu/eventing-rabbitmq/blob/0e5db67084c5ea98020819ff6332710ce88f2fbe/Makefile#L279-L280) if it's not set, so rather than defaulting to `kind` it defaults to `eventing-rabbitmq-e2e`.

**Comment by ChunyiLyu** [comment]
@benmoss I can run conformance tests and e2e tests locally now and they are all passing after running necessary make commands. What should we do with this PR? Would it be fine to merge it now or should we wait til this is fixed: https://github.com/knative-sandbox/eventing-rabbitmq/pull/577

**Comment by benmoss** [comment]
I'd ask @gabo1208

**Pull request merged**